doc/new-guidelines.md

```diff
@@ -90,22 +133,27 @@ their styles in the top-level of the properties map, prefer to add them inside t
  ]
 ```
 
-### Always apply animated styles in the style file
+Also its fine to keep one liner styles in view
+
+```clojure
+;; ok
+[rn/view {:flex 1 :padding-top 5}]
+```
+
+
+### Never apply animated styles in the style file
 
-When implementing styles for reanimated views, we should always define
-them in the style file and apply animations with reanimated/apply-animations-to-style
-in the style definition.
 
 ```clojure
-;; bad
+;; good
 (defn circle
   []
   (let [opacity (reanimated/use-shared-value 1)]
     [reanimated/view {:style (reanimated/apply-animations-to-style
                               {:opacity opacity}
                               style/circle-container)}]))
 
-;; good
+;; bad
```

Comment:
I personally don't agree with this. Like that we would be mixing styles with view. Animation styles are still styles. There are animation styles that have 5+ style props, I don't think we should be having these in the view file.

Reply:
but its not like static styles right? its not about that you open styles ns and change style, its about defining which styles will be animated (dynamic) if you don't like map in your view you always can move it in a separate function with no problem, if you have 5+ style props it will look like `(style/circle-container something-prop1 something-prop2 something-prop3 something-prop4 something-prop5 )` so it doesn't look good anyway, and better to have somethingl like 

```clojure
[ra/view {:style (ra/styles styles/static animated-map)}
```

Reply:
At first I was on @flexsurfer's camp, then a comment from @briansztamfater convinced me to move to @OmarBasem camp, and I was cool there ;) I think styles in general don't look good anyway, but I tend to agree with Omar a little bit more.

Reply:
could you pls share comment from @briansztamfater

Reply:
> I think styles in general don't look good anyway

could you elaborate?

Reply:
i feel bad merging without @OmarBasem and @briansztamfater approves :(

Reply:
@flexsurfer sorry I somewhat skipped this PR, will check it now :)

Reply:
Yeah, so we discussed this some time ago, then the discussion leaded to update the guideline as it currently is. The link to the last time we defined this is https://github.com/status-im/status-mobile/pull/15489#discussion_r1149898021

In general terms, I agree with @OmarBasem and @ilmotta. One of the problems of not declaring animated styles in the styles file, is that it would make our components to have lot of boilerplate code for merging styles with animated styles, when we can abstract the responsibility to the style definition in the styles file and make our components look cleaner.
Also we could end up maintaining a same style for the same view on two different files, making it a bit more difficult to maintain and understand.
Regarding the naming, in the past I proposed renaming `apply-animations-to-style` to `merge-style`, the reason behind this is here https://github.com/status-im/status-mobile/pull/15275#issuecomment-1462182376. But I assume that is another topic

In conclusion I don't agree with this guideline change but I can adapt, the important thing IMO is to have a guideline to follow and make it immutable at least for a long time :P

Reply:
Also, regarding his example

> but its not like static styles right? its not about that you open styles ns and change style, its about defining which styles will be animated (dynamic) if you don't like map in your view you always can move it in a separate function with no problem, if you have 5+ style props it will look like `(style/circle-container something-prop1 something-prop2 something-prop3 something-prop4 something-prop5 )` so it doesn't look good anyway, and better to have somethingl like
> 
> ```clojure
> (let [opacity (reanimated/use-shared-value 1)
>        ra-styles  {:opacity opacity}]
>     [ra/view (ra/apply-styles style/static ra-styles)])
> ```

I feel that as more complex or big the animated style is, the more complex the component will be. And if we have many reanimated views in the same component, it might read worse.

 ```clojure
 (let [opacity (reanimated/use-shared-value 1)
       width (reanimated/use-shared-value 100)
       ra-styles {:opacity opacity
                  :width   width
                  :height  width}]
     [ra/view {:style (ra/apply-styles style/static ra-styles)}])
```
vs
 ```clojure
 (let [opacity (reanimated/use-shared-value 1)
       width (reanimated/use-shared-value 100)]
     [ra/view {:style (style/square opacity width)}])
```

Reply:
I don't like  the idea of having animated styles in the `style`  namespace, but it's true that doing this makes the view code more clean and easier to read, so I vote for having them in the `style` ns.